util/build_configs/cray-internal/setenv-xc-x86_64.bash

```diff
@@ -266,7 +268,19 @@ else
     log_debug "with config=$BUILD_CONFIGS_CALLBACK"
 
     # Exit immediately to skip (avoid building) unwanted Chapel configs
-
+    
```

Comment:
Could use a comment describing why we're skipping pic support for other configs.